From 6cc5055d8ab56b6f6045a6e85c0a7173a315c67c Mon Sep 17 00:00:00 2001
From: ArchanaSinde1 <archana.shinde2504@gmail.com>
Date: Fri, 31 Mar 2023 07:23:53 +0000
Subject: [PATCH] Relax dependencies

---
 sdks/python/setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sdks/python/setup.py b/sdks/python/setup.py
index 7525177a5b..c1f108abeb 100644
--- a/sdks/python/setup.py
+++ b/sdks/python/setup.py
@@ -140,7 +140,7 @@ except ImportError:
 if sys.platform == 'win32' and sys.maxsize <= 2**32:
   pyarrow_dependency = ''
 else:
-  pyarrow_dependency = 'pyarrow>=3.0.0,<10.0.0'
+  pyarrow_dependency = 'pyarrow>=3.0.0,<=11.0.0'
 
 # We must generate protos after setup_requires are installed.
 def generate_protos_first():
@@ -188,7 +188,7 @@ if __name__ == '__main__':
     # restrict the protobuf with tight upper bound(3.19.4)
     protobuf_dependency = ['protobuf>3.12.2,<3.19.5']
   else:
-    protobuf_dependency = ['protobuf>3.12.2,<4']
+    protobuf_dependency = ['protobuf>=4.21.1,<4.23.0']
 
   setuptools.setup(
       name=PACKAGE_NAME,
-- 
2.34.1

