diff -Naur apache-beam-2.40.0-org/setup.py apache-beam-2.40.0/setup.py
--- apache-beam-2.40.0-org/setup.py	2022-08-08 09:28:45.897621001 -0400
+++ apache-beam-2.40.0/setup.py	2022-08-26 06:12:15.017589531 -0400
@@ -124,7 +124,7 @@
 if sys.platform == 'win32' and sys.maxsize <= 2**32:
   pyarrow_dependency = ''
 else:
-  pyarrow_dependency = 'pyarrow>=0.15.1,<8.0.0'
+  pyarrow_dependency = 'pyarrow>=0.15.1,<=10.0.0'
 
 # We must generate protos after setup_requires are installed.
 def generate_protos_first():
@@ -217,7 +217,7 @@
         'grpcio>=1.33.1,<2',
         'hdfs>=2.1.0,<3.0.0',
         'httplib2>=0.8,<0.21.0',
-        'numpy>=1.14.3,<1.23.0',
+        'numpy>=1.14.3,<1.24.0',
         'pymongo>=3.8.0,<4.0.0',
         'protobuf>=3.12.2,<4',
         'proto-plus>=1.7.1,<2',
