From d7531234bd471336afd4e449acdee3d64131926a Mon Sep 17 00:00:00 2001
From: ArchanaSinde1 <archana.shinde2504@gmail.com>
Date: Fri, 31 Mar 2023 05:12:07 +0000
Subject: [PATCH] Relax dependencies

---
 sdks/python/setup.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sdks/python/setup.py b/sdks/python/setup.py
index c4c7f106c3..d872e69bac 100644
--- a/sdks/python/setup.py
+++ b/sdks/python/setup.py
@@ -140,7 +140,7 @@ except ImportError:
 if sys.platform == 'win32' and sys.maxsize <= 2**32:
   pyarrow_dependency = ''
 else:
-  pyarrow_dependency = 'pyarrow>=0.15.1,<10.0.0'
+  pyarrow_dependency = 'pyarrow>=0.15.1,<=11.0.0'
 
 # We must generate protos after setup_requires are installed.
 def generate_protos_first():
@@ -188,7 +188,7 @@ if __name__ == '__main__':
     # restrict the protobuf with tight upper bound(3.19.4)
     protobuf_dependency = ['protobuf>3.12.2,<3.19.5']
   else:
-    protobuf_dependency = ['protobuf>3.12.2,<4']
+    protobuf_dependency = ['protobuf>=4.21.1,<4.23.0']
 
   setuptools.setup(
       name=PACKAGE_NAME,
@@ -248,7 +248,7 @@ if __name__ == '__main__':
         'grpcio>=1.33.1,!=1.48.0,<2',
         'hdfs>=2.1.0,<3.0.0',
         'httplib2>=0.8,<0.21.0',
-        'numpy>=1.14.3,<1.23.0',
+        'numpy>=1.14.3,<1.24.0',
         # Tight bound since minor version releases caused breakages.
         'objsize>=0.5.2,<0.6.0',
         'pymongo>=3.8.0,<4.0.0',
-- 
2.34.1

