diff --git a/setup.py b/setup1.py
index 559f1f7..737396d 100644
--- a/setup.py
+++ b/setup1.py
@@ -140,8 +140,8 @@ except ImportError:
 if sys.platform == 'win32' and sys.maxsize <= 2**32:
   pyarrow_dependency = ''
 else:
- # pyarrow_dependency = 'pyarrow>=0.15.1,<=10.0.0'
-  pyarrow_dependency = 'pyarrow>=0.15.1,<10.0.0'
+  pyarrow_dependency = 'pyarrow>=0.15.1,<=10.0.0'
+#  pyarrow_dependency = 'pyarrow>=0.15.1,<10.0.0'
 
 # We must generate protos after setup_requires are installed.
 def generate_protos_first():
@@ -249,8 +249,8 @@ if __name__ == '__main__':
         'grpcio>=1.33.1,!=1.48.0,<2',
         'hdfs>=2.1.0,<3.0.0',
         'httplib2>=0.8,<0.21.0',
-  #      'numpy>=1.14.3,<1.24.0',
-        'numpy>=1.14.3,<1.23.0',
+        'numpy>=1.14.3,<1.24.0',
+  #      'numpy>=1.14.3,<1.23.0',
         # Tight bound since minor version releases caused breakages.
         'objsize>=0.5.2,<0.6.0',
         'pymongo>=3.8.0,<4.0.0',
